tests/src/FunctionalJavascript/WebformCivicrmTestBase.php

```diff
@@ -64,4 +65,66 @@ protected function setUp() {
     ]);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function initFrontPage() {
+    parent::initFrontPage();
+    // Fix hidden columns on build page.
+    $this->getSession()->resizeWindow(1440, 900);
+  }
+
+  /**
+   * Configures contact information fieldset into its own wizard page.
+   */
+  protected function configureContactInformationWizardPage() {
+    $this->drupalGet($this->webform->toUrl('edit-form'));
+
+    // Add the "Contact information" wizard page.
+    $this->getSession()->getPage()->clickLink('Add page');
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $element_form = $this->getSession()->getPage()->findById('webform-ui-element-form-ajax');
+    $element_form->fillField('Title', 'Contact information');
+    $this->assertSession()->waitForElementVisible('css', '.machine-name-value');
+    $element_form->pressButton('Save');
+    $this->assertSession()->assertWaitOnAjaxRequest();
+
+    // Put contact elements into new page.
+    $contact_information_page_row_handle = $this->assertSession()->elementExists('css', '[data-drupal-selector="edit-webform-ui-elements-contact-information"] a.tabledrag-handle');
+    // Move up twice to be the top-most element.
+    $this->sendKeyPress($contact_information_page_row_handle, 38);
+    $this->sendKeyPress($contact_information_page_row_handle, 38);
+    $contact_information_page_row_handle->blur();
+    $contact_fieldset_row_handle = $this->assertSession()->elementExists('css', '[data-drupal-selector="edit-webform-ui-elements-civicrm-1-contact-1-fieldset-fieldset"] a.tabledrag-handle');
+    $this->sendKeyPress($contact_fieldset_row_handle, 39);
+    $contact_fieldset_row_handle->blur();
+    $this->getSession()->getPage()->pressButton('Save elements');
+    $this->assertSession()->assertWaitOnAjaxRequest();
+  }
+
+  /**
+   * Send a key press to an element.
+   *
+   * @var \Behat\Mink\Element\NodeElement $element
+   *   The element.
+   * @var string|int $char
+   *   The character or char key code
+   * @var string $modifier
+   *   The modifier (could be 'ctrl', 'alt', 'shift' or 'meta').
+   */
+  protected function sendKeyPress(NodeElement $element, $char, $modifier = '') {
```

Comment:
🙌 now we can reuse this to send up/down/left/right keys on the Build form for re-arranging elements.